controllers/nginx/configuration.md

```diff
@@ -340,22 +342,23 @@ The next table shows the options, the default value and a description
 |use-gzip|"true"|
 |use-http2|"true"|
 |vts-status-zone-size|10m|
-|worker-processes|<number of CPUs>|
+|worker-processes|number of CPUs|
```

Comment:
I'm not sure about these either. It was on my list to ask about for later.

I have a couple more changes in the pipeline and I don't see a clear way to express:

```
|ssl-dh-param|value provided by OpenSSL|
```

Your suggestion would fix this case though and perhaps that is good enough for now? Another idea is to simply write `&lt;` instead of `<`.

Some of these values are quoted but others are not. Perhaps we could quote the items to be taken literally *or* `&lt;`/`&gt;` the values that are not?